ketcher-autotests/tests/File-Management/Open-And-Save-Files/Open-File/Load-Fragment/load-fragment.spec.ts

```diff
@@ -0,0 +1,205 @@
+    const yForMoveElement = 180;
+    await dragMouseTo(xForMoveElement, yForMoveElement, page);
+
+    await openFileAndAddToCanvas('Rxn-V2000/reaction-3.rxn', page);
+  });
+
+  test('Load Fragment - Load as fragment (Add to Canvas) - smi-files', async ({
+    page,
+  }) => {
+    /*
+     * Test case: EPMLSOPKET-1861
+     */
+    await openFileAndAddToCanvas('Molfiles-V2000/glutamine.mol', page);
+    await openFileAndAddToCanvas('SMILES/1840-cyclopentyl.smi', page);
+  });
+
+  test('Open file - Input SMILE-string', async ({ page }) => {
+    /*
+     * Test case: EPMLSOPKET-1836
+     */
+    const smileString = 'C1=CC=CC=C1';
+    const secondSmileString = 'C1=CC=CC=C1>>C1=CC=CC=C1';
+    const thirdSmileString = 'CCCCCC';
+    const incorrectSmileString = 'CCCCC0';
+
+    await openPasteFromClipboard(page, smileString);
+    await waitForLoad(page, async () => {
+      await pressButton(page, 'Cancel');
+    });
+
+    await openPasteFromClipboard(page, smileString);
+    await waitForLoad(page, async () => {
+      await pressButton(page, 'Open as New Project');
+    });
+    await takeEditorScreenshot(page);
+
+    await openPasteFromClipboard(page, secondSmileString);
+    await waitForLoad(page, async () => {
+      await pressButton(page, 'Open as New Project');
+    });
+    await takeEditorScreenshot(page);
+
+    await openPasteFromClipboard(page, thirdSmileString);
+    await waitForLoad(page, async () => {
+      await pressButton(page, 'Open as New Project');
+    });
+    await takeEditorScreenshot(page);
+
+    await openPasteFromClipboard(page, incorrectSmileString);
+    await pressButton(page, 'Open as New Project');
+    const convertErrorMessage = await page
+      .getByTestId('info-modal-body')
+      .textContent();
+    const expectedErrorMessage =
+      'Convert error!\nGiven string could not be loaded as (query or plain) molecule or reaction, see the error messages: ' +
+      "'molecule auto loader: SMILES loader: cycle number 0 is not allowed', " +
+      "'scanner: BufferScanner::read() error', 'scanner: BufferScanner::read() error', " +
+      "'molecule auto loader: SMILES loader: cycle number 0 is not allowed'";
+    expect(convertErrorMessage).toEqual(expectedErrorMessage);
+  });
+
+  test('Open file - Open *.ket file', async ({ page }) => {
+    /*
+     * Test case: EPMLSOPKET-2933
+     */
+    await openFileAndAddToCanvas('Molfiles-V2000/glutamine.mol', page);
+    await openFileAndAddToCanvas('KET/simple-chain.ket', page);
+  });
+
+  test('Open/Import structure as a KET file - create KET file', async ({
+    page,
+  }) => {
+    /*
+     * Test case: EPMLSOPKET-4702 (creating file)
+     */
+    const { x, y } = await getCoordinatesOfTheMiddleOfTheScreen(page);
+
+    await selectAtomInToolbar(AtomButton.Hydrogen, page);
+    await clickInTheMiddleOfTheScreen(page);
+    await resetCurrentTool(page);
+
+    await moveMouseToTheMiddleOfTheScreen(page);
+    const shiftForHydrogen = 150;
```

Comment:
Perhaps we should put all the constants at the beginning of the test?

Reply:
Done